projects/storefrontapp-e2e-cypress/cypress/helpers/accessibility/tabbing-order.ts

```diff
@@ -21,7 +21,9 @@ export function checkElement(tabElement: TabElement) {
       return;
     }
     case TabbingOrderTypes.GENERIC_INPUT: {
-      cy.focused().should('have.attr', 'type', 'text');
+      cy.focused()
```

Comment:
we don't check anything here. we could focus any input and it would have value === ''. can we do it better?